client/src/components/Results/Cards.js

```diff
@@ -239,15 +241,35 @@ class Card extends React.Component {
                     : null}
                 </div>
               </div>
-              <div className="card-content">
-                {textMap.parsedCOVID === "CLOSED DUE TO COVID" ? (
-                  <div className="covid-item">CLOSED</div>
-                ) : (
-                  textMap.parsedHours
-                )}
-              </div>
+
+            
+                 <div className="card-header">
+            <div className="card-category">{textMap.parsedCategory}</div>
```

Comment:
Here is that repeated category.  Really this whole block should be removed.

Reply:
Done